plugins/database/src/db_plugin.cpp

```diff
@@ -799,7 +793,12 @@ static int _delColl( rsComm_t *rsComm, collInfo_t *collInfo ) {
         return CATALOG_NOT_CONNECTED;
     }
 
-    splitPathByKey( collInfo->collName, logicalParentDirName, MAX_NAME_LEN, logicalEndName, MAX_NAME_LEN, '/' );
+    if (const auto ec = splitPathByKey(collInfo->collName, logicalParentDirName, MAX_NAME_LEN, logicalEndName, MAX_NAME_LEN, '/'); ec < 0) {
+        irods::log(LOG_NOTICE, fmt::format(
```

Comment:
just poking to make sure you meant LOG_NOTICE.

Reply:
I didn't feel that the message would be particularly helpful, but maybe this does rise to the level of `LOG_ERROR`... willing to change. `splitPathByKey` emits error messages of its own, so I figured that would be enough for the rare case that something does go wrong. Thoughts?

Reply:
later in the commit it was treated more loudly - that is all.    i think NOTICE is probably fine, as you said, since it emits its own error messages.

Reply:
Oooh, I see. Yes, I will go ahead and make these `LOG_ERROR` then. I don't expect this situation to come up very much so I suppose we ought to know about it when it does.

Reply:
cool cool.